### PR TITLE
Fixes gh-2059 implement ability to run multiple iproto threads

### PR DESCRIPTION
**Pull request by pgulutzan**

### CODE REVIEW

**Comment by NickVolynkin** [comment]
@EvgenyMekhanik please have a look.

**Review by NickVolynkin — APPROVED** [review]

File: doc/reference/configuration/cfg_networking.rst [hunk 1]
```diff
@@ -69,3 +69,21 @@
     | Type: integer
     | Default: 16320
     | Dynamic: **yes**
+
+.. _cfg_networking-iproto_threads:
+
+.. confval:: iproto_threads
```
Comment (NickVolynkin) [inline doc/reference/configuration/cfg_networking.rst:75]:
```suggestion
..   confval:: iproto_threads
```

This was indented with 2 spaces and the text below with 4. As a result, the text is formatted as a blockquote. See https://tarantool-develop.herokuapp.com/en/doc/pgulutzan-gh-2059/reference/configuration/#confval-iproto_threads.

Comment (pgulutzan) [reply doc/reference/configuration/cfg_networking.rst:75]: I did make a mistake but it doesn't seem to be with confval, which is done the same way that other items in the file are done. However, the text below was was indented with 5 spaces.

Comment (NickVolynkin) [reply doc/reference/configuration/cfg_networking.rst:75]: Oh, it's 5 spaces, indeed. Should be 4 as well.

**Pull request merged**